### PR TITLE
migration of daemon reports data from redshift to RDS. 

### PR DESCRIPTION
**Pull request by prashantramangupta**
Daemon makes a call to API  with service statistics data. Upon successful authorization of the token, we process the data and store it in RDS.

### CODE REVIEW

File: platform-usage-authorizer/lambda_handler.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            print("Processing [" + str(path) + "] with body [" + str(payload_dict) + "]")
+            token_instance = Token()
+            data = token_instance.validate_token(daemon_id=payload_dict['x-daemonid'],
+                                                 token=payload_dict['x-token'])
+            print(data)
+        else:
+            return get_response(500, "Invalid URL path.")
+        response = get_lambda_authorizer_response_format(event=event, allow=data.get('validated', False))
+        return response
+    except Exception as e:
```
Comment (raamb) [inline platform-usage-authorizer/lambda_handler.py]: Lets integrate slack handler

File: platform-usage-authorizer/lambda_handler.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        data = None
+        if "/event" == path:
+            payload_dict = event['headers']
+            print("Processing [" + str(path) + "] with body [" + str(payload_dict) + "]")
+            token_instance = Token()
+            data = token_instance.validate_token(daemon_id=payload_dict['x-daemonid'],
+                                                 token=payload_dict['x-token'])
+            print(data)
+        else:
+            return get_response(500, "Invalid URL path.")
```
Comment (raamb) [inline platform-usage-authorizer/lambda_handler.py]: return get_lambda_authorizer

Comment (prashantramangupta) [reply platform-usage-authorizer/lambda_handler.py]: resolved

File: platform-usage-authorizer/lambda_handler.py (hunk 1 above)
Comment (prashantramangupta) [reply platform-usage-authorizer/lambda_handler.py]: resolved

**Pull request merged**